Sources/SATSCore/Components/NoticeView/NoticeModifier.swift

```diff
@@ -0,0 +1,83 @@
+import SwiftUI
+
+/// Internal modifier, it should only be used via the `View.inlineNotice` method
+struct NoticeModifier: ViewModifier {
+    @Binding var notice: Notice?
+    let edge: Notice.Edge
+
+    private let transitionDuration: TimeInterval = 0.2
+    private let autoDismissTime: TimeInterval = 3
```

Comment:
Hmm.. I feel 3 seconds is a bit short. I think we should increase this to 4 or even 5 maybe.